src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    const [property, value] = style.split(':', 2)
```

Comment:
The split operation is limited to 2 parts, which is good for separating the property and value. However, consider handling cases where there might be a colon in the value itself, such as in a URL or content property.